template/views/authentication/config/index.js

```diff
@@ -26,7 +26,7 @@ const session = require("express-session");
 const MongoStore = require("connect-mongo");
 
 // Connects the mongo uri to maintain the same naming structure
-const MONGO_URI = require("../utils/consts");
+const MONGO_URI = process.env.MONGODB_URI || "mongodb://localhost:27017/{{name}}";
```

Comment:
why are we repeating this variable instead of putting it in the const file?

Reply:
Main benefits of a constants file is having a central place for managing globally used values. Setting an extra constants file on top of the .env file seems unnecessary for the existing lessons and perhaps make the boilerplate more complicated to understand and navigate.